front/src/applications/editor/Editor.tsx

```diff
@@ -47,7 +47,11 @@ const Editor = () => {
   const dispatch = useAppDispatch();
   const navigate = useNavigate();
   const { openModal, closeModal } = useModal();
-  const { updateInfraID, selectLayers } = useOsrdActions() as EditorSliceActions;
+  const {
+    updateInfraID,
+    selectLayers,
+    updateMapSettings: updateMapSettingsActions,
```

Comment:
You only have 1 action here ;)
```suggestion
    updateMapSettings: updateMapSettingsAction,
```